### PR TITLE
French localization for account UI strings

### PR DESCRIPTION
**Pull request by DimitriDR**
Hi, here my proposal for the French localization for account UI strings. I'm sure it's not (still perfect), I'll adjust small details with use if I see any weird stuff.

### CODE REVIEW

**Comment by jonkoops** [comment]
Hi @DimitriDR, thanks for your PR. Could you get your changes reviewed by another French native speaker? We always like to make sure there is at least one person reviewing that knows the subject matter.

**Comment by jonkoops** [comment]
@Paul-Van-Uytvinck could I ask you to review this PR?

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  "errorRemovedMessage": "Impossible de supprimer {{userLabel}} à cause de : {{error}}",
+  "errorSignOutMessage": "Impossible de se déconnecter : {{error}}",
+  "expires": "Expire",
+  "filterByName": "Filtrer par nom...",
+  "firstName": "Prénom",
+  "fullName": "{{givenName}} {{familyName}}",
+  "groupDescriptionLabel": "Voir les groupes auxquels vous êtes associé",
+  "groups": "Groupes",
+  "hasAccessTo": "A accès à",
+  "infoMessage": "En cliquant sur Supprimer l'accès, vous supprimerez les permissions accordées de cette application. Cette application ne pourra plus utiliser vos informations.",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:63]:
```suggestion
  "infoMessage": "En cliquant sur Supprimer l'accès, vous supprimerez les permissions accordées à cette application. Cette application ne pourra plus utiliser vos informations.",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  "internalApp": "Interne",
+  "inUse": "Utilisé",
+  "ipAddress": "Adresse IP",
+  "lastAccessedOn": "Dernier accès",
+  "lastName": "Nom de famille",
+  "link": "Lier le compte",
+  "linkedAccounts": "Comptes liés",
+  "linkedAccountsIntroMessage": "Gérew les connexion via les comptes tiers.",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:71]:
```suggestion
  "linkedAccountsIntroMessage": "Gérez les connexions via les comptes tiers.",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  "refreshPage": "Rafraichir la page",
+  "removeAccess": "Supprimer l'accès",
+  "removeConsentError": "Impossible de supprimer le consentement en raison de : {{error}}",
+  "removeConsentSuccess": "Consentement supprimé",
+  "removeCred": "Supprimer {{0}}",
+  "removeModalMessage": "Cela supprimera l'autorisation d'accès actuellement accordée pour {{0}}. Vous devrez de nouveau obtenir les accès si vous souhaitez utiliser cette application.",
+  "requestor": "Demandeur",
+  "required": "Requis",
+  "resourceAlreadyShared": "La ressource est déjà partagée avec cet utilisateur.",
+  "resourceIntroMessage": "Partagez votre ressource avec vos membres de l'équipe",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:105]:
```suggestion
  "resourceIntroMessage": "Partagez votre ressource avec les membres de votre équipe",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  "sharedWithMe": "Partagé avec moi",
+  "shareError": "Impossible de partager la ressource à cause de : {{error}}",
+  "shareSuccess": "La ressource a bien été partagée.",
+  "shareTheResource": "Partager la ressource - {{0}}",
+  "shareUser": "Ajouter les utilisateurs avec qui partager la ressource",
+  "shareWith": "Partager avec ",
+  "signedInDevices": "Appareils connectés",
+  "signedInDevicesExplanation": "Déconnectez-vous de tout appareil non familier.",
+  "signedOutSession": "Déconnecté {{0}}/{{1}}",
+  "signingIn": "Connexoin",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:125]:
```suggestion
  "signingIn": "Connexion",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+  "signingInDescription": "Configuez les façons de vous authentifier.",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:126]:
```suggestion
  "signingInDescription": "Configurez les façons de vous authentifier.",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+  "tryAgain": "Réessayer",
+  "two-factor": "Authentification à deux facteurs",
+  "unknownOperatingSystem": "Système d'exploitation inconnu",
+  "unknownUser": "Anonyme",
+  "unLink": "Dissocier le compte",
+  "unlinkedEmpty": "Aucun fournisseur non lié",
+  "unlinkedLoginProviders": "Fournisseurs de connexion non liés",
+  "unLinkError": "Impossible de dissocier à cause de : {{error}}",
+  "unLinkSuccess": "Compte dissocié avec succès",
+  "unShare": "Tout ne plus partager",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:150]:
```suggestion
  "unShare": "Ne plus rien partager",
```

File: js/apps/account-ui/public/locales/fr/translation.json [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+  "unShareAllConfirm": "Êtes-vous sûr de vous complétement supprimer tous les partages ?",
```
Comment (Paul-Van-Uytvinck) [inline js/apps/account-ui/public/locales/fr/translation.json:151]:
```suggestion
  "unShareAllConfirm": "Êtes-vous sûr de vouloir complétement supprimer tous les partages ?",
```

**Review by Paul-Van-Uytvinck — CHANGES_REQUESTED** [review]

**Review by DimitriDR — COMMENTED** [review]
Super useful typo fixes by @Paul-Van-Uytvinck

**Comment by jonkoops** [comment]
@Paul-Van-Uytvinck can I ask you to take another look and confirm everything looks good now? If so, I think we can go ahead and merge this in.

**Review by Paul-Van-Uytvinck — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
I've set this to automatically merge once the CI passes. Thanks for the contribution @DimitriDR and @Paul-Van-Uytvinck!

**Pull request merged**